server/migrations/20201211080408-attachment-no-cascade.js

```diff
@@ -0,0 +1,37 @@
+const tableName = 'attachments';
```

Comment:
Remove cascade on delete association as attachment<->document relationships are not 1-1 in reality, this was a mistake.